src/main/scala/io/iohk/ethereum/network/PeerActor.scala

```diff
@@ -267,15 +266,6 @@ class PeerActor(
       messageSubscribers = messageSubscribers.filterNot(_.ref == sender())
   }
 
-  def handlePeerChainCheck(rlpxConnection: RLPxConnection): Receive = {
```

Comment:
I think, it's not needed anymore since we have `FastUploadHost`, is that right ( @ntallar ? )